docker-compose.yml

```diff
@@ -38,3 +38,8 @@ services:
     volumes:
       - ./solrdata:/opt/solr/server/solr/data/
     restart: always
+    
+  mailhog:
+    image: yappabe/mailhog
```

Comment:
Is there any specific reason for using yappabe/mailhog? It is a quite "big" container, 251 MB compressed. F.ex. the mailhog/mailhog is only 7 MB compressed.